src/js/providers/tracks-mixin.js

```diff
@@ -573,8 +573,12 @@ function _removeCues(renderNatively, tracks) {
             // Cues are inaccessible if the track is disabled. While hidden,
             // we can remove cues while the track is in a non-visible state
             // Set to disabled before hidden to ensure active cues disappear
-            track.mode = 'disabled';
-            track.mode = 'hidden';
+            if (track.mode !== 'disabled') {
```

Comment:
"Cues are inaccessible if the track is disabled"

That's why we need to change the mode to "hidden", so that we can access the cues and remove them.

Would this check work if we tested for the track being hidden instead?
`if (track.mode !== 'hidden')`

Reply:
What Rob said.

If it's just `'disabled'` that causes Edge to barf, then I'd suggest:

```
if (track.mode !== 'disabled') {
    track.mode = 'disabled';
}
track.mode = 'hidden';
```

Reply:
That didn't work. I don't know what the bug is but setting it to disabled is not the problem. It's setting to hidden while disabled

Reply:
@robwalch I tried that and it didn't work. The state was disabled when entering that block, not hidden

Reply:
yeah we already talked through this.